hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java

```diff
@@ -191,8 +192,7 @@ public BinaryResource getAvatar(String agentId)
             ensureCorrectHttpStatusReturned(SC_OK, httpResponse);
             log.atDebug().log("Successfully retrieved avatar for Agent with id: {}", agentId);
 
-            String filePrefix = format("avatar-%s-%s", agentId, System.currentTimeMillis());
-            File tempImageFile = TempFileProvider.createTempFile(httpResponse.body(), filePrefix, "png");
+            File tempImageFile = TempFileProvider.createTempFile(httpResponse.body(), UUID.randomUUID().toString(), "png");
```

Comment:
It would still be ideal to add a prefix to the uuid so that these temp files can easily be identified e.g.: via `ls avatar-*`

Reply:
That's right, I accidentally deleted that. Thanks!